compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -1722,40 +1722,33 @@ unsupported {} from `{}` with element `{}` of size `{}` to `{}`"#,
         }
 
         enum Style {
-            Float,
-            Int(/* is signed? */ bool),
+            Float(/* width */ u64),
+            Int(/* is signed? */ bool, /* width */ u64),
+            Pointer,
             Unsupported,
         }
 
-        let (in_style, in_width) = match in_elem.kind() {
+        let determine_style = |kind: &ty::TyKind<'_>| match kind {
             // vectors of pointer-sized integers should've been
```

Comment:
this comment is out-of-date -- it should probably just be removed

Reply:
Good point, removed